contrib/test-suite/test-suite.yml

```diff
@@ -10,3 +10,46 @@
   - ci-build-debug/test-suite.log
   - ci-build-debug/ci-mock-result/*.log
   timeout: 6 hours
+
+- name: Multihost tests Tier 0
+  machines:
+  - client
+  tasks:
+  - name: Execute tests
+    shell: |
```

Comment:
I would propose to move this shell function into a script (the same way it is currently done with intg tests)

Reply:
Why would you like to do that? This is sssd-test-suite and ci specific. To run the tests manually you just need to call pytest (see readme).

Reply:
Just to prevent this .yml file getting messy.

Reply:
Ok, this also allowed me to use variables from distro.sh so it should be now clear what are the unsupported platforms.